src/testsuite/vec_pwr9_dummy.c

```diff
@@ -0,0 +1,165 @@
+/*
+ Copyright (c) [2017] IBM Corporation.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+
+ vec_pwr9_dummy.c
+
+ Contributors:
+      IBM Corporation, Steven Munroe
+      Created on: April 27, 2018
+ */
+
+//#pragma GCC push target
+#pragma GCC target ("cpu=power9")
+
+#include <vec_int128_ppc.h>
+
+vui8_t
+__test_absdub_PWR9 (vui8_t __A, vui8_t __B)
+{
+  return vec_absdub (__A, __B);
+}
+
+vui128_t /*__attribute__ ((__target__ ("cpu=power9")))*/
```

Comment:
shall we delete the commented attribute here, since you have the global attribute above?